spec/services/id_token_builder_spec.rb

```diff
@@ -61,22 +61,41 @@
       expect(decoded_payload[:nonce]).to eq(identity.nonce)
     end
 
+    context 'it sets the vot' do
+      it 'sets the vot' do
+        identity.vot = 'Pb'
+        expect(decoded_payload[:vot]).to eq('C1.C2.P1.Pb')
+      end
+    end
+
     context 'it sets the acr' do
       context 'ial2 request' do
+        before do
+          identity.ial = 2
+          identity.acr = Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF
```

Comment:
I think a test where we include an AAL ACR value alongside an IAL once may be wise to cover the concerns I describe in my comment above ☝️